molgenis-api-tests/pom.xml

```diff
@@ -51,6 +51,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>${awaitility.version}</version>
```

Comment:
Remove version tag and add dependency to dependency management section in parent pom.xml to assert that the correct dependency is used for all tests.